Vester/Public/Invoke-Vester.ps1

```diff
@@ -70,8 +70,7 @@ function Invoke-Vester {
     .LINK
     https://github.com/WahlNetwork/Vester
     #>
-    [CmdletBinding(SupportsShouldProcess = $true,
-                   ConfirmImpact = 'Medium')]
+
```

Comment:
This is not unused. It passes `-WhatIf` support through to `VesterTemplate.Tests.ps1`